app/uk/gov/hmrc/merchandiseinbaggagefrontend/model/core/Declaration.scala

```diff
@@ -148,6 +154,7 @@ case class DeclarationJourney(sessionId: SessionId,
                               maybeValueWeightOfGoodsExceedsThreshold: Option[YesNo] = None,
                               goodsEntries: GoodsEntries = GoodsEntries.empty,
                               maybeNameOfPersonCarryingTheGoods: Option[Name] = None,
+                              maybeEmailAddress: Option[Email] = None,
```

Comment:
I'd expect a change to the declaration spec to show that the declaration can't be complete (and so you can't go to /check-your-answers) unless the email address has been provided